app/shared/src/translations/__tests__/Translate.test.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+
+import { NativeModules } from 'react-native';
+
+import Translate from '../Translate';
+
+NativeModules.RNTranslationManager = {
+  translate: jest.fn(key => key),
+};
```

Comment:
Sorry, one last thing - shouldn't this be part of the `jestSetup` script?

Reply:
Thanks. It will actually happen very soon because `Text` component will require the translations...